src/inlineExpression.tsx

```diff
@@ -171,8 +171,8 @@ export function InlineRenderer({ value }: { value?: string }): JSX.Element {
   }
 
   // Explicitly render text/plain
-  const preferredMimeType = rendermime.preferredMimeType(mimeBundle ?? {});
-  if (preferredMimeType === 'text/plain') {
+  const preferred = rendermime.preferredMimeType(mimeBundle ?? {}, 'any');
```

Comment:
JupyterLab uses the `trusted` concept to choose renderers here:

```suggestion
  const preferred = rendermime.preferredMimeType(mimeBundle ?? {}, trusted ? 'any' : 'ensure');
```

I think we should too?

Reply:
I think you are maybe right here, it just isn't actually what I think we want? This will revert to text for widgets when not "trusted" explicitly, which is really weird.

The notebook seems to selectively "trust" code cells as you run them and/or actually only trusts the code cells, it is very strange to me that a code cell can show a widget, but a markdown cell cannot unless the notebook is trusted?

Thoughts on which path to take?!

Reply:
There are a few more docs here on this:
https://jupyter-server.readthedocs.io/en/stable/operators/security.html#our-security-model

I don't think that we can tap into the output trust model at the moment, as we are in a markdown cell and that does not expect to have outputs?

Reply:
Hmm, although that guide says that Markdown cells are never trusted, that doesn't appear to be true; with this patch applied I see the widgets after trusting and re-executing the notebook.